### ISSUE TITLE
Broken "publisher" example

### ISSUE THREAD

**Issue by jaques-sam** [opentok-rust/gst-plugin-opentok#7]
When running the publisher example, you'll see the following error: `didn't accept URI`

❯ GST_DEBUG=3 GST_PLUGIN_PATH=/gst-plugin-opentok/target/debug/ cargo run --example publisher -- --api-key $TOKBOX_API_KEY --session-id <hided> --remote
    Finished dev [unoptimized + debuginfo] target(s) in 0.15s
     Running `target/debug/examples/publisher --api-key 47760411 --session-id <hided> --token <hided> --remote`
***** OpenTok Linux SDK | Version/Revision: linux_llvm-2.25.0/d4431ca9bcd213d5f2faa19289fa919cfeac7e97 | Library built on: Apr  3 2023 | CA bundle version: Tue Jan 10 04:12:06 2023 GMT | Copyright 2023 Vonage | https://www.vonage.com/legal/communications-apis/terms-of-use/
***** OpenTok Linux SDK | Version/Revision: linux_llvm-2.25.0/d4431ca9bcd213d5f2faa19289fa919cfeac7e97 | Library built on: Apr  3 2023 | CA bundle version: Tue Jan 10 04:12:06 2023 GMT | Copyright 2023 Vonage | https://www.vonage.com/legal/communications-apis/terms-of-use/
0:00:00.189793458 111786 0x367e0048f800 WARN                 GST_URI gsturi.c:675:gst_element_make_from_uri: opentoksinkremote0 didn't accept URI 'opentok-remote://<hided>?key=<hided>&token=<hided>': OpenTokRemoteLaunchFailed
Error! Missing element opentoksink

Furthermore the last error is not really correct and is the only thing you'll see without setting GST_DEBUG.

**Comment by philn** [comment]
As mentioned in the README of opentok-rs: `For now the only supported platform is Linux and 2.19.1 is the only supported version.`

You seem to use version 2.25?

**Comment by jaques-sam** [comment]
Yeah, I updated to see if there is a difference. It's the same with 2.19.

     Running `target/debug/examples/publisher --api-key <hidden> --session-id <hidden> --token <hidden> --remote`
*********************** OpenTok Linux SDK ****************************
**** Version/Revision: linux_llvm-2.19.1/7b24b80eb36da92f16c39e3cfee55fd2b4eef6aa
**** Library built on: Apr 15 2021
**** Copyright 2021 TokBox, Inc.
**** Licensed under the Apache License, Version 2.0
****
0:00:00.120586159 64268 0x557ecc663b30 WARN          adaptivedemux2 gstadaptivedemuxelement.c:41:adaptivedemux2_base_element_init: Failed to load libsoup library
0:00:00.121416809 64268 0x557ecc663b30 WARN          adaptivedemux2 gstadaptivedemuxelement.c:41:adaptivedemux2_base_element_init: Failed to load libsoup library
0:00:00.122139761 64268 0x557ecc663b30 WARN          adaptivedemux2 gstadaptivedemuxelement.c:41:adaptivedemux2_base_element_init: Failed to load libsoup library
0:00:00.274008052 64268 0x557ecc663b30 WARN            GST_REGISTRY gstregistrybinary.c:552:gst_registry_binary_check_magic: Binary registry magic version is different : 1.3.0 != 1.12.0
0:00:00.276342933 64268 0x557ecc663b30 WARN                 GST_URI gsturi.c:676:gst_element_make_from_uri: opentoksinkremote0 didn't accept URI 'opentok-remote://<hidden>?key=<hidden>&token=<hidden>': OpenTokRemoteLaunchFailed
Error! Missing element opentoksink.

Maybe it's related to https://github.com/opentok-rust/opentok-rs/issues/3.

**Comment by philn** [comment]
Error! Missing element opentoksink.

did you build it? is GST_PLUGIN_PATH correctly set? when you run `gst-inspect-1.0 opentoksink` do you see anything? is `ldd path/to/libgstopentok...so` reporting not-found libraries?

**Comment by jaques-sam** [comment]
`gst-inspect-1.0 opentoksink` works, it gives me the plugin details.
`ldd` shows all libs as existing.

But this is normal as you can see `WARN GST_URI gsturi.c:676:gst_element_make_from_uri: opentoksinkremote0 didn't accept URI 'opentok-remote://?key=&token=': OpenTokRemoteLaunchFailed`

`Error! Missing element opentoksink.` just seems like the wrong error message...
Can you try it out? Is fairly fast to see.

Could it be that the stream-id became necessary in the uri?
Another piste could be: https://github.com/opentok-rust/opentok-rs/issues/3.

**Comment by philn** [comment]
Sorry I'm not involved in this project anymore. Maybe @thiblahute can help you.
